agent/exec/containerd/adapter.go

```diff
@@ -362,21 +362,16 @@ func (c *containerAdapter) shutdown(ctx context.Context) error {
 		return errAdapterNotPrepared
 	}
 
-	if c.exitStatus == nil {
-		var err error
-		c.log(ctx).Debug("Deleting")
-
-		tasks := c.client.TaskService()
-		rsp, err := tasks.Delete(ctx, &execution.DeleteRequest{ContainerID: c.name})
-		if err != nil {
-			c.log(ctx).WithError(err).Debug("Task.Delete failed")
-			return err
-		}
-		c.log(ctx).Debugf("Task.Delete success, status=%d", rsp.ExitStatus)
-		c.exitStatus = makeExitError(rsp.ExitStatus, "")
-	} else {
-		c.log(ctx).Debug("Task already deleted, error=%s", c.exitStatus)
+	var err error
```

Comment:
It does, looks like it is there in master and this is just moving it around, it actually disappears entirely later in the series, in `containerd: Use Task.Delete() in adapter.shutdown()`.

A downside which I hadn't anticipated of reviewing commit by commit is that GH marks comments such as this as "Outdated" right as they are made, so they are collapsed in the webui and I only notice them via the email notification.